### ISSUE TITLE
I can't install Erlide despite trying everything

### ISSUE THREAD

**Issue by hungdhbg** [erlang/erlide_eclipse#358]
I can't install Erlide despite trying everything @vladdu

**Comment by hungdhbg** [comment]
> Have you tried to follow https://erlide.org/articles/eclipse/120_Installing-and-updating.html ? Where does it fail, and what errors do you get?

The problem is that Erlide's server has been turned off. So it won't be possible to install it

**Comment by hungdhbg** [comment]
Yes, I used the link "https://erlide.org/update" but when eclipse downloads part of it, it will say it cannot connect to the server, and will not be able to continue.

> Are you using `https://erlide.org/update` as described in the docs?
> 
> Eclipse downloads the file https://erlide.org/update/p2.index when installing and it seems downloadable.
> 
> A clean installation using Eclipse 2024-09 (4.33.0) worked here when testing now.


Yes, I used the link "https://erlide.org/update" but when eclipse downloads part of it, it will say it cannot connect to the server, and will not be able to continue. This is the error I am referring (even though I only selected IDE Tool settings):
(![Screenshot 2024-10-23 181422](https://github.com/user-attachments/assets/3b24e890-4506-453b-9117-5b9d028ff0d3))

**Comment by bjosv** [comment]
Ah, ok. My eclipse installation (via [snap](https://snapcraft.io/install/eclipse/ubuntu)) provides [org.apache.commons.jxpath_1.3.0.jar](http://download.eclipse.org/eclipse/updates/4.34-I-builds/I20241017-2140/plugins/org.apache.commons.jxpath_1.3.0.jar) which is why it works here..

**Comment by bjosv** [comment]
I managed to reproduce your problem on a Windows installation using Eclipse 2024-09 (4.33).
They seem to have changed the latest Eclipse a bit compared to previous versions, and it seem to use JEE.

If you can use the previous released Eclipse version instead, i.e. [2024-06 (4.32)](https://www.eclipse.org/downloads/packages/release/2024-06/r)
then you could try-out the prerelease of Erlide 0.62.0, which contains a fix to be able to run it on newer Eclipse.

Install it according to the [docs](https://erlide.org/articles/eclipse/120_Installing-and-updating.html), but use the URL `https://erlide.org/update/prerelease` instead,
and only install `erlide - Erlang development tools`.

**Comment by hungdhbg** [comment]
> I managed to reproduce your problem on a Windows installation using Eclipse 2024-09 (4.33). They seem to have changed the latest Eclipse a bit compared to previous versions, and it seem to use JEE.
> 
> If you can use the previous released Eclipse version instead, i.e. [2024-06 (4.32)](https://www.eclipse.org/downloads/packages/release/2024-06/r) then you could try-out the prerelease of Erlide 0.62.0, which contains a fix to be able to run it on newer Eclipse.
> 
> Install it according to the [docs](https://erlide.org/articles/eclipse/120_Installing-and-updating.html), but use the URL `https://erlide.org/update/prerelease` instead, and only install `erlide - Erlang development tools`.

Thank you sir, i made it
